shared/login/register/code-page/dumb.js

```diff
@@ -69,6 +71,7 @@ const enterTextMock = {
 const dumbComponentMap: DumbComponentMap<Render> = {
   component: Render,
   mocks: {
+    'Normal': baseMock,
```

Comment:
This is currently not passing tests because `baseMock` doesn't have an `otherDeviceRole` value. Perhaps add a default one to it?